raytracing/imagingpath.py

```diff
@@ -693,14 +693,16 @@ def displayRange(self, axes=None):
         """
 
         displayRange = self.largestDiameter
-        if displayRange == float('+Inf') or displayRange <= 2 * self.objectHeight:
-            displayRange = 2 * self.objectHeight
+        objHeight = abs(self.objectHeight)
```

Comment:
If it is never valid, then we should create a @property getter and setter to prevent using a negative value in the first place.